generator/source/index.md

```diff
@@ -1,24 +1,36 @@
 ---
 hide: true
 layout: page
+title: Hyperledger Grid
 permalink: /
 feature-img: "images/hero-bg.jpg"
 # Copyright (c) 2018 Bitwise IO, Inc.
 # Licensed under Creative Commons Attribution 4.0 International License
 # https://creativecommons.org/licenses/by/4.0/
 ---
 
-# Hyperledger Grid
+Hyperledger Grid is a platform for building supply chain solutions that include
+distributed ledger components. It provides a set of libraries, data models, and
+SDKs for developing supply chain smart contracts and client interfaces.
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+This project's goal is to accelerate development of modular blockchain-based
+solutions for cross-industry supply chain problems. Hyperledger Grid provides
+domain-specific data models, such as GS1 product definitions that can be used in
+a product catalog, example smart contracts, and other modular components.
 
-# Project Status
+* [Learn about Hyperledger Grid](/about)
+
+* [Read the documentation](/docs)
+
+* [Get the code](https://github.com/hyperledger/grid)
+
+* [Join the community](/community)
+
+* [See Hyperledger Grid in the
+  news](https://www.hyperledger.org/category/hyperledger-grid)
+
+Hyperledger Grid is an open source project on GitHub. We welcome contributors,
+both organizations and individuals, to help shape project direction, contribute
+ideas, provide use cases, and work on specific tools and examples.
+Please [join the conversation](/community).
 
-This Hyperledger project is in _incubation_. It was proposed to the community
```

Comment:
Should we remove this indication of its status of "in incubation"?

Reply:
My commit already removes that line.  However, I was planning to move this info to the About page, under "Project Status", with a link to the Hyperledger Grid project page.

Reply:
I restored this paragraph to the home page.